sdk/python/packages/flet-core/src/flet_core/geolocator.py

```diff
@@ -257,55 +318,23 @@ async def open_location_settings_async(
         )
         return opened == "true"
 
-    def service_enable(self, wait_timeout: Optional[float] = 10) -> bool:
-        opened = self.invoke_method(
-            "service_enable",
-            wait_for_result=True,
-            wait_timeout=wait_timeout,
-        )
-        return opened == "true"
```

Comment:
So, `service_enable` and `service_disable` were redundant?

Reply:
Somehow...
The enable was meant to start the position-stream while disable was meant to stop/cancel it.

Now, streaming is automatically started if position change callback is provided.